autograder/cli/submission/common.py

```diff
@@ -5,9 +5,9 @@
 import autograder.util.file
 import autograder.util.gzip
 
-def output_grading_result(result, base_dir = '.'):
+def output_grading_result(result, base_dir = '.', keep_dir = False):
```

Comment:
Despite our email conversation, is this actually necessary?
Since `out_dir` is the directory removed (and that already has the submission id on it), we may not have to worry about deleting it.
Remember, our use case is someone getting attempts for the same user from multiple assignments.

I think an issue here is actually using `id` vs `short-id`.
For fetch/submissions, id makes sense, but I think short-id is better in this case.
So instead, we may want to make an argument that controls that.
Don't be afraid to just update the semantics of this function and update the usage in fetch/submissions.

For fetch/submissions, we want:
```
out_dir
 - <full submission id>
 - <full submission id>
 - ...
```

And for fetch/attempts, we want:
```
out_dir
 - <assignment id>
     - <email>
         -  <short submission id>
         -  <short submission id>
         - ...
```